src/Chapter1.hs

```diff
@@ -615,7 +630,9 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 n = lastdigit (abs n) + lastdigit (div (abs n) 10)
```

Comment:
First `abs` in here is redundant, since you already do `abs`. Otherwise looks good :+1:

Reply:
Just some additional info. We were asking to use local variables in this task, and there are actually a few ways to do so. Your solution is valid, but for your info you could do it alternatively in the following way:

* Specify local variables via `let-in` and reuse already defined `lastDigit` from a previous task
* You might be interested in knowing that there's a function called `divMod` which returns a pair of results `div` and `mod` at the same time. It is supposed to be a bit more efficient, than calling `div` and `mod` several times. So you can use it as well :slightly_smiling_face: